client/src/components/SeasonCollectionImageCardComponent/seasoncollectionimg.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import './seasoncollectionimg.css'
+import imageFile1 from "./Image 1.svg"
+import imageFile2 from "./Image 2.svg"
+
+
+// need to work here
+let season = "Fall";
+let year = "2020";
+
+const SeasonCollectionImageCardComponent = (props) =>
+
+    <div>
+
+        <div className="seasonCollectionSquare">
+            <div className="seasonCollectionImgContainer">
```

Comment:
Can you make all class names lowercase and separated by dashes? I.e. `season-collection-square`